packages/core/src/components/overlay/overlay.tsx

```diff
@@ -172,8 +178,14 @@ export class Overlay extends React.Component<IOverlayProps, IOverlayState> {
             </CSSTransitionGroup>
         );
 
+        const mergedClassName = classNames({
+            [Classes.OVERLAY]: true,
```

Comment:
Yeah realized that after opening the PR but forgot to change it. Good catch. Done. 👍